src/components/Transcript/Transcript.js

```diff
@@ -1,33 +1,226 @@
 import React from 'react';
+import cx from 'classnames';
```

Comment:
Is there a particular reason to use this library instead of using vanilla JS to build conditional classnames? I have not used this library before, so I was curious.

Reply:
`classnames ` is purely for ergonomics. It helps avoid long ternaries or akward interpolation when you have multiple conditional classnames. It [weighs virtually nothing](https://bundlephobia.com/package/classnames@2.5.1) but can be removed for vanilla JS, I probably just included it out of habit.

Reply:
That sounds good. I think there are multiple places in the code, where we use conditional classnames with long conditions. And I think this could be a nice addition to the code, which will improve the readability and maintenance of the code going forward 👍